psamm/commands/tableexport.py

```diff
@@ -62,16 +62,31 @@ def run(self):
 
     def _reaction_export(self):
         property_set = set()
-        for j in self._model.parse_reactions():
-            property_set.update(j.properties)
+        for reaction in self._model.parse_reactions():
+            property_set.update(reaction.properties)
 
         property_list_sorted = sorted(
             property_set, key=lambda x: (x != 'id', x != 'equation', x))
 
-        print('\t'.join([text_type(x) for x in property_list_sorted]))
-        for i in self._model.parse_reactions():
-            print('\t'.join(text_type(i.properties.get(property))
-                            for property in property_list_sorted))
+        print('\t'.join(
+            [text_type(x) for x in property_list_sorted] + ['in_model']))
+
+        if self._model.has_model_definition():
+            model_reactions = set(self._model.parse_model())
+            for reaction in self._model.parse_reactions():
+                line_content = [text_type(reaction.properties.get(property))
+                                for property in property_list_sorted]
+                if reaction.id in model_reactions:
+                    line_content.append('True')
+                else:
+                    line_content.append('False')
+                print('\t'.join(line_content))
```

Comment:
Yeah, but this is easy to modify as there is no need to call `model.create_metabolic_model()`, but the next one for compounds would be first check if there is a model definition and then create the model, and then it would still not be so elegant.